### DISCUSSION TITLE
Aa torque problem with Android auto

### DISCUSSION THREAD

**Discussion by alfreddo56** [agronick/aa-torque#84] [General]
Hi, I'm Paolo After updating to Android, AA Torque Performance Monitor is not shown in the Android Auto app list, what should I do to make it work again? Thank you

**Comment by 1cmoody23** [comment]
Im having the saame problem, app is installed, I set developer option in AA, I set unknown source, In about under AA, it lists AAtorque as one of the packages but I dont have a button to launch it in the car screen.

**Reply by danjustus** [reply]
I don't even see AAtorque listed in the apps in AA. How do I get it there?

**Comment by agronick** [comment]
Google is locking down on unknown apps. One way to get around it is to get an AAWireless dongle and enable developer mode.
